emoji-copy@felipeftn/emojiOptionsBar.js

```diff
@@ -1,24 +1,20 @@
-const St = imports.gi.St;
+import * as St from 'gi://St';
 
 /* Import the current extension, mainly because we need to access other files */
-const ExtensionUtils = imports.misc.extensionUtils;
-const Me = ExtensionUtils.getCurrentExtension();
-const Extension = Me.imports.extension;
+import * as Extension from './extension.js';
 
-/* Stuffs for translations etc. */
-const textDomain = imports.gettext.domain('emoji-copy');
-const getText = textDomain.gettext;
+import { gettext as _ } from 'resource:///org/gnome/shell/extensions/js/extensions.js';
```

Comment:
This line [needs](https://gjs.guide/extensions/upgrading/gnome-shell-45.html#extensionutils) to be changed to
```js
import {gettext as _} from 'resource:///org/gnome/shell/extensions/extension.js';
```

Note that not only the `/js` part from the path is removed, but the filename `extension.js` is now also in singular instead of plural.